src/output-json.c

```diff
@@ -345,9 +348,19 @@ int OutputJSONBuffer(json_t *js, LogFileCtx *file_ctx, MemBuffer *buffer)
         return TM_ECODE_OK;
 
     SCMutexLock(&file_ctx->fp_mutex);
-    if (json_out == ALERT_SYSLOG) {
-        syslog(alert_syslog_level, "%s", js_s);
-    } else if (json_out == ALERT_FILE || json_out == ALERT_UNIX_DGRAM || json_out == ALERT_UNIX_STREAM) {
+    if (json_out == ALERT_SYSLOG) 
+	{
+		if (syslog_cee_format == TRUE)
```

Comment:
That is my mistake. I stood up a new VM environment for pull requests. The spaces/tab setting must have been off in my vimrc.